_posts/2018-05-17-self-organization.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+title: "Self Organization"
+date: 2018-05-17 12:0:00 +0200
+categories: motivation how-we-learn
+author_name: Ugurcan Sengit
+author_url : /author/ugurcansengit
+author_avatar: ugurcansengit
+read_time : 8
+feature_image: posts/2018-04-20-holidaycheck-apprenticeship/apprenticeship.png
+---
+
+I remember the very first day as an apprentice at [HolidayCheck](https://twitter.com/holidaychecklab). Honestly, it's only four months ago, not so hard to recall. The thing is, it feels like it's been a lifetime already. All the things I've learnt during the apprenticeship... One just can't put a price tag on them. Time management and self-organization made them happen.
+
+At the very first day, I was both excited and afraid. Because I've never had six months of free space to improve myself after starting my professional career. The first challenge for me was to shape these six months. I've needed to create a structure and follow it strictly in order not to get lost during my journey. After my apprenticeship half-time presentation, I think I've nailed it, that's why I write this blog post to share my process with you.
```

Comment:
What do you think about changing the order of the first sentence? "I was both excited and afraid the very first day". 
It's a stylistic choice, so it's fine either way.

Reply:
On the second sentence, the "Because" is not necessary, if you don't mind removing it. The sentence retains its meaning. 

Also, consider changing "I've" to "I'd" as in "I had never had six months of space to improve myself". I can't say why, but my mind tells me it's better that way.

Reply:
You could change "The first challenge for me" to "My first challenge". Less words 😄

Reply:
You can replace "I've needed to create" with "I had to create". Retains meaning, contains less words. 

Also, at the end of the sentence, "in order not to get lost" should be "in order to not get lost".

Reply:
A period after "I think I've nailed it." instead of the current comma,  gives it better structure, and allows the paragraph to end in its own sentence.

Reply:
Other than "in order not to", I've updated as you suggested. Thanks! "In order not to" is also valid for that sentence.